Documentation/pt-br/README.md

```diff
@@ -0,0 +1,39 @@
+# Documentação
+
+Quick te ajuda a verificar como programas em Swift e Objective-C se comportam.
+
+Fazê-lo de forma eficaz não é apenas questão de saber como usar Quick, no entanto. Os guias nesse diretório podem te ajudar a escrever testes eficazes --não apenas usando Quick, até mesmo XCTest ou outros framework de teste.
+
+Cada guia cobre um tópico em particular. Se você é completamente novo a teste unitário, considere lê-los na ordem que são introduzidos abaixo:
+
+- **[Configurar Testes no seu Xcode Project](SettingUpYourXcodeProject.md)**:
+  Leia isto se você esta com problemas para usar o código da sua aplicação de dentro dos seus arquivos de teste.
+- **[Testes Eficazes Usando XCTest: Arrange, Act e Assert](ArrangeActAssert.md)**:
+  Leia isto para aprender como escrever `XCTestcase` testes que o ajudarão a escrever código mais rápido e mais eficaz.
+- **[Não Teste o Código, ao Invés Verifique Comportamento](BehavioralTesting.md)**:
```

Comment:
I feel that "Não Teste o Código, Verifique Comportamento" feels more natural, but that may be just me 😆